<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>361</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(11, 0, 57);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="usertype_label">
        <property name="styleSheet">
         <string notr="true">background-color:rgb(4, 255, 0);color:rgb(117, 2, 2);font-size:12pt;</string>
        </property>
        <property name="text">
         <string>Operator</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">background-color:rgb(0, 157, 255);color:rgb(212, 255, 70);font-size:18pt;</string>
        </property>
        <property name="text">
         <string>Oversight of current products in the making</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 237)</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTableWidget" name="tw_tasks">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(240, 234, 255)</string>
        </property>
        <column>
         <property name="text">
          <string>Task</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Workplace</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Given to</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>at</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>deprecated</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="tw_employees">
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(237, 255, 239)</string>
        </property>
        <column>
         <property name="text">
          <string>Employee</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QPushButton" name="giveTask">
        <property name="styleSheet">
         <string notr="true">color: rgb(149, 38, 30); background-color: rgb(241, 224, 255);</string>
        </property>
        <property name="text">
         <string>Give Task</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="btn_close">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(244, 242, 255)</string>
        </property>
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="btn_task_by_hand">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(244, 242, 255)</string>
        </property>
        <property name="text">
         <string>Give Task by Hand</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
